mockgcp/mockasset/testdata/savedquery/crud/script.yaml

```diff
@@ -0,0 +1,13 @@
+- pre: |
+    rm -f /tmp/query.yaml
+    echo "IamPolicyAnalysisQuery:" > /tmp/query.yaml
```

Comment:
Suggest change the tmp file name to be resource specific in case it deletes other files.